content/docs/user-guide/dvc-files.md

```diff
@@ -143,9 +143,12 @@ stages:
     metrics:
       - performance.json
   training:
-    desc: Training stage description
-    cmd: python train.py
+    desc: Train model with Python
+    cmd:
+      - pip install -r requirements.txt
+      - python train.py --out ${model_file}
     deps:
+      - requirements.txt
```

Comment:
This is being reintroduced after rolling it back accidentally in https://github.com/iterative/dvc.org/pull/2053/files#diff-584bff5a435baee84112ffda33bc2d9dc94e1512e7306322f43655d5e7593876

Reply:
In any case, this isn't a great example. What is the main use case for multiple commands in a single stage (sharing deps and outs)? For now I've asked in https://github.com/iterative/dvc/issues/4373#issuecomment-751830121

Reply:
let's not complicate the base example then for now

Reply:
OK I'm all about letting this be for now. But just FYI, there's some discussion on this question in https://github.com/iterative/dvc/issues/4373#issuecomment-751996212.

My conclusions are that:
- This is mainly syntax sugar as an alt. to `&&` (which could be quite nice in some cases like setting up a bunch of env vars at once).
- Another good use could be to add "parallel" stages in a single definition: while not the cleanest dvc.yaml code, it would save extra lines.
- If neither of those examples seem like a strong/real user need for this, it could be un-documented for simplicity.

Reply:
so, there were some good points and example in that discussion (e.g. `cd`) - should we take a real example?

Reply:
OK looking in https://github.com/search?q=filename%3Advc.yaml+%26%26&type=code ⌛

Reply:
any updates here?

Reply:
I'm not prioritizing this part for the current PR (per https://github.com/iterative/dvc.org/pull/2062#discussion_r549473711), but will continue searching for an actual real life example ASAP.

Reply:
let's not touch it then make it a separate issue?

Reply:
Oh I thought that by "base example" you meant the multi-cmd one with requirements.txt, which was already merged to master but I accidentally rolled it back at some point. But yes, I can extract this... ⌛

Reply:
I see,  `but I accidentally rolled it back at some point.` - that's where all this confusion comes from. Up to you then 👍

Reply:
Already extracted to #2070 🙂 resolving here